source/brailleHelpers.py

```diff
@@ -0,0 +1,26 @@
+# A part of NonVisual Desktop Access (NVDA)
+# This file is covered by the GNU General Public License.
+# See the file COPYING for more details.
+# Copyright (C) 2021 NV Access Limited
+
+from typing import Optional
+
+import config
+from controlTypes import DescriptionFrom
+
+
+def _descriptionToDisplay(
+		description: Optional[str],
+		descriptionFrom: Optional[DescriptionFrom],
+) -> Optional[str]:
+	""" Determines if the given description should be reported. Returns None if description should not be
+	reportedm otherwise returns the description to report.
+	"""
```

Comment:
```suggestion
	""" Determines if the given description should be reported. Returns None if description should not be
	reported, otherwise returns the description to report.
	"""
```

Reply:
I'm going to drop this helper function. DRY is good, but this is only used twice, and those usages haven't been around long, we also don't want hasty abstractions.

Reply:
This makes sense. I was hoping to reduce the complexity of the functions which use it, and generally `braille.py` further (or at least prevent from adding to it). But that can wait for a more focused change on `braille.py`.